propel/resource_metric.go

```diff
@@ -467,7 +491,7 @@ func expandMetricFilters(def []interface{}) ([]*pc.FilterInput, diag.Diagnostics
 		f := &pc.FilterInput{
 			Column:   filter["column"].(string),
 			Operator: pc.FilterOperator(filter["operator"].(string)),
-			Value:    filter["value"].(string),
+			Value:    filter["value"].(*string),
 		}
 
```

Comment:
The new schema has make optional this field, it is because we added the new filter option `IS_NULL` and `IS_NOT_NULL`.

Just doing this change makes the plugin crash, to fix it you need:
1. change from `required` to `optional` the `value` field in the Filter, line 92
2. see the suggestion

```suggestion
		}
		
		if def, ok := filter["value"]; ok {
			value := def.(string)
			f.Value = &value
		}


```